examples/agents-integrations/agno-agents/.env.example

```
@@ -0,0 +1,8 @@
+GOOGLE_API_KEY= 
+LLM_PROVIDER=gemini
+LLM_MODEL=gemini/gemini-2.5-flash
+LLM_API_KEY=
+EMBEDDING_PROVIDER=fastembed
+EMBEDDING_MODEL=jinaai/jina-embeddings-v2-base-en
+EMBEDDING_DIMENSIONS=768
+QDRANT_URL=http://localhost:6333
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)